Makefile

```diff
@@ -118,6 +118,13 @@ lint: golangci-bin ## Run configured golangci-lint and pre-commit.sh linters aga
 	testbin/golangci-lint run ./... --config=./.golangci.yaml
 	hack/pre-commit.sh
 
+.PHONY: create-rdr-env
+create-rdr-env: ## Create a new rdr environment.
+	./hack/dev-env.sh create
+
+destroy-rdr-env: ## Destroy the existing rdr environment.
+	./hack/dev-env.sh destroy
```

Comment:
These targets would be nice, but since we cannot hide the virtual env, it will be better to implement them like this:

    cd test
    drenv start envs/regional-dr.yaml

Again this will not affect the current directory of the caller.

Reply:
I decided to retain the script even though the virtual environment creation and activation is outside of it now. This is mainly because I would like to bring in MDR related code in this script and having the prefixes for the environments in this script is better than the Makefile.